valhalla/midgard/openlr.h

```diff
@@ -0,0 +1,144 @@
+#ifndef VALHALLA_MIDGARD_OPENLR_H_
+#define VALHALLA_MIDGARD_OPENLR_H_
+
+#include "pointll.h"
+#include "base64.h"
+
+namespace valhalla {
+namespace midgard {
+
+namespace OpenLR {
+
+// Specify 1-byte padding for this struct
+#pragma pack(push, 1)
+struct TwoPointLinearReference  {
+    using byte = std::uint8_t;
+
+    // Byte 0
+    byte unused_1 : 1;
+    bool ArF1 : 1;
+    bool no_point : 1;
+    bool ArF0 : 1;
+    bool has_attributes : 1;
+    byte version  : 3;
+
+    // Note: wire-format is big-endian
+    // Bytes 1-6
+    std::int32_t fixed_longitude : 24;
```

Comment:
Vahalla uses uint8_t, int16_t, uint64_t etc. throughout the code base. No real reason to prefix with std.